assets/js/blocks/handpicked-products/block.json

```
@@ -0,0 +1,74 @@
+{
+    "name": "woocommerce/handpicked-products",
+    "version": "1.0.0",
```

looking at blocks in Gutenberg I think we should use `apiVersion: 2` here and also if we include a reference to block.json schema we'll have autocomplete and hints on some of the fields! 

All blocks in Gutenberg start with $schema and apiVersion followed by name - we could follow this pattern.

```suggestion
    "$schema": "https://schemas.wp.org/trunk/block.json",
    "apiVersion": 2,
    "name": "woocommerce/handpicked-products",
```

---

we'd benefit from having this included everywhere where we moved to block.json so far I think

---

The `apiVersion` is already at the end of the json file, not sure if the `version` number is correct thought 🤔 
I'll include the `$schema`.

---

ah, missed it sorry, ah, so the version is the version of the block? what is it?

---

got the answer from schema :) 

> The current version number of the block, such as 1.0 or 1.0.3. It’s similar to how plugins are versioned. This field might be used with block assets to control cache invalidation, and when the block author omits it, then the installed version of WordPress is used instead.